Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -617,52 +623,72 @@ class MessageDetailViewController: OWSViewController, UIScrollViewDelegate {
             owsFail("\(TAG) Missing messageTextProxyView")
             return
         }
+        guard let scrollView = scrollView else {
+            owsFail("\(TAG) Missing scrollView")
+            return
+        }
+        guard let contentView = contentView else {
+            owsFail("\(TAG) Missing contentView")
+            return
+        }
+        guard let bubbleView = bubbleView else {
+            owsFail("\(TAG) Missing bubbleView")
+            return
+        }
+        guard let bubbleSuperview = bubbleView.superview else {
+            owsFail("\(TAG) Missing bubbleSuperview")
+            return
+        }
         guard let messageTextTopConstraint = messageTextTopConstraint else {
-            owsFail("\(TAG) Missing messageTextProxyView")
+            owsFail("\(TAG) Missing messageTextTopConstraint")
             return
         }
         guard let messageTextHeightLayoutConstraint = messageTextHeightLayoutConstraint else {
-            owsFail("\(TAG) Missing messageTextProxyView")
+            owsFail("\(TAG) Missing messageTextHeightLayoutConstraint")
             return
         }
         guard let messageTextProxyViewHeightConstraint = messageTextProxyViewHeightConstraint else {
-            owsFail("\(TAG) Missing messageTextProxyView")
+            owsFail("\(TAG) Missing messageTextProxyViewHeightConstraint")
             return
         }
-        guard let scrollView = scrollView else {
-            owsFail("\(TAG) Missing scrollView")
-            return
-        }
-        guard let contentView = contentView else {
-            owsFail("\(TAG) Missing contentView")
+        guard let bubbleViewWidthConstraint = bubbleViewWidthConstraint else {
+            owsFail("\(TAG) Missing bubbleViewWidthConstraint")
             return
         }
 
         if messageTextView.width() != messageTextProxyView.width() {
             owsFail("\(TAG) messageTextView.width \(messageTextView.width) != messageTextProxyView.width \(messageTextProxyView.width)")
         }
 
+        let bubbleViewHMargin: CGFloat = 10
```

Comment:
Can we dedupe these to use the equivalent constants extracted around line 310

Reply:
Removed!